### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by adam50200** [pkgxdev/pkgx#875] [Help] [answered]
what and how should I do here? is there a guide? I don't understand. I want to score a lot of points. retweets of what to do give links.

**Comment by adam50200** [comment]
ignore it?

**Comment by adam50200** [comment]
I don’t understand what should I do

**Reply by jhheider** [reply]
what are specifically trying to do? if you want to contribute to the project, the easiest way is to package some software for pkgxdev/pantry. if you're looking to get started using `pkgx`, take a look at [docs.pkgx.sh](https://docs.pkgx.sh/).

**Comment by adam50200** [comment]
is there a video guide on what to do on this site?

**Comment by GlenLamb** [comment]
https://www.youtube.com/watch?v=iv8rSLsi1xo

**Comment by adam50200** [comment]
where and how can I attach my discord to my github account?

**Reply by michaelessiet** [reply]
You can't. You can only do that if you would like to enable advanced github features like creating repositories, discussions and issues on third party platforms. May I know why you want to connect you discord to your github account?

**Reply by michaelessiet** [reply]
Oh, nvm. You go to discord settings and click on "connections" and click on the github logo

**Reply by adam50200** [reply]
I want to get the contributor role

**Comment by jperica** [comment]
I do not see any help here. I could connect my discord to my github.

**Comment by adam50200** [comment, marked as answer]
![B08A94BB-D681-4D49-986D-4A4F9EA50EEB](https://github.com/pkgxdev/pkgx/assets/133254982/9d5bfc8c-c51c-49cf-990b-80cfccd31ae1)

right? then why don't I have a contributor role?

**Reply by michaelessiet** [reply]
Hi Adam, for you to get the contributor role you need have contributed to one of the pkgx repos, either by identifying key issues or by opening PRs and getting them merged
